Essentials/src/main/java/com/earth2me/essentials/Settings.java

```diff
@@ -943,8 +943,28 @@ public long getAutoAfk() {
     }
 
     @Override
-    public long getAutoAfkKick() {
-        return config.getLong("auto-afk-kick", -1);
+    public long getAutoAfkKick(final User user) {
```

Comment:
Can a user belong to multiple sets? If so, I would think that the most generous limit should be used. If not, you can return after the first limit is found (and eliminate the variable `limit`).